src/graphics/program-lib/programs/standard.js

```diff
@@ -1445,12 +1462,16 @@ var standard = {
                 code += "   addDirLightMap();\n";
             }
 
+
+            if (hasAreaLights){
+                code += "   float roughness = max((1.0 - dGlossiness) * (1.0 - dGlossiness), 0.001);\n";
```

Comment:
@mvaligursky Is this calculation of roughness correct? I took this out of one of the chunks but would be good to verify

Reply:
This is the glossiness to roughness conversion I use for the GGX specular implementation (which is a model that is not widely used yet) - its based off what I heard was used a few years back in Frostbite - I was planning to revisit and recalibrate this if/when we switch using roughness.